internal/factsengine/gatherers/hostsfile_test.go

```diff
@@ -0,0 +1,101 @@
+package gatherers
```

Comment:
Could we have other test where the provided argument is not found? to test the abnormal path of the lookup

Reply:
It's already indirectly tested in the `TestHostsFileIgnoresCommentedHosts` test, but I can add a specific test as I noticed it may be not 100% clear.

Reply:
In the end... I've decided not to add it, as it's basically a 1:1 copy of that tests. Let me know if it's ok